### ISSUE TITLE
Bad `Requested Python version (X.Y) not installed` output

### ISSUE THREAD

**Issue by jurko-gospodnetic** [tox-dev/tox#692]
When running `tox` (at least on Windows) and there is and there is a test environment defined in the `tox` configuration file for which a suitable Python interpreter version is not installed on the system, tox output incudes stderr output like `Requested Python version (X.Y) not installed`.

The error described above occurs even when just running `tox` with the `--help` command-line option, and even if explicitly requesting a different test environment using the `-e` command-line option.

For example, when running `tox --help` with the current `tox` master (20215c1ee1891cefa29f69651ef85263feb4472b) with only Python 2.7 & 3.6 installed, I get the following output:

```
(venv) D:\Workplace\Open-source\Python tox>tox --help
Requested Python version (3.4) not installed
Requested Python version (3.5) not installed
usage: tox [--version] [-h] [--help-ini] [-v] [--showconfig] [-l] [-a]
           [-c CONFIGFILE] [-e envlist] [--notest] [--sdistonly]
           [--installpkg PATH] [--develop] [-i URL] [--pre] [-r]
           [--result-json PATH] [--hashseed SEED] [--force-dep REQ]
           [--sitepackages] [--alwayscopy] [--skip-missing-interpreters]
           [--workdir PATH]
           [args [args ...]]
...
```

**Comment by jurko-gospodnetic** [comment]
Pull request #686 added to address the issue.

**Comment by gaborbernat** [comment]
Resolved by #686
